agent/eni/netlinkWrapper/mocks/mock_netlinkWrapper.go

```diff
@@ -0,0 +1,52 @@
+// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
you can add a file to auto generate this like: https://github.com/aws/amazon-ecs-agent/blob/master/agent/engine/dockerclient/generate_mocks.go